toonz/sources/translations/update.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# Usage: update.sh LANGUAGE
+
+BASE_DIR=$(cd `dirname "$0"`; pwd)
+TOONZLANG=$1
+
+[ ! -d "${BASE_DIR}/${TOONZLANG}" ] || mkdir -p "${BASE_DIR}/${TOONZLANG}"
```

Comment:
Should this be 
`[ ! -d "${BASE_DIR}/${TOONZLANG}" ] && mkdir -p "${BASE_DIR}/${TOONZLANG}"` 
or
`[ -d "${BASE_DIR}/${TOONZLANG}" ] || mkdir -p "${BASE_DIR}/${TOONZLANG}"` 
?

Reply:
Fixed now. Thank you!